multimedia/v4l2rtspserver/patches/001_live555_sprintf_format

```diff
@@ -0,0 +1,20 @@
+--- a/live/liveMedia/ServerMediaSession.cpp
++++ b/live/liveMedia/ServerMediaSession.cpp
```

Comment:
There is missing commit header. Who is author of this patch, why this is necessary. Is this backport? If not, will it be upstreamed?

Reply:
I am the author of the patch, live555 is crashing using openwrt toolchain. I will add header.